pypeit/spectrographs/vlt_xshooter.py

```diff
@@ -943,12 +953,54 @@ def init_meta(self):
         """
         super().init_meta()
         # Add the name of the dispersing element
-        # dispangle and filter1 are not defined for Shane Kast Blue
 
         # Required
         self.meta['decker'] = dict(ext=0, card='HIERARCH ESO INS OPTI3 NAME')
 
-    def order_platescale(self, binning = None):
+    @property
+    def norders(self):
+        """
+        Number of orders observed for this spectograph.
+        """
+        return 11
+
+    @property
+    def order_spat_pos(self):
+        """
+        Return the expected spatial position of each echelle order.
+
+        from pypeit import edgetrace
+        edges = edgetrace.EdgeTraceSet.from_file('MasterEdges_A_1_DET01.fits.gz')
+
+        nrm_edges = edges.edge_fit[edges.nspec//2,:] / edges.nspat
+        slit_cen = ((nrm_edges + np.roll(nrm_edges,1))/2)[np.arange(nrm_edges.size//2)*2+1]
+        """
+        # This starts by ignoring the first, partial order (25?)
+        #  Order 24 is very faint and not included here
+        #  Order 12 is very faint and not included here (the flat crashes out with issues..)
+        return np.array([0.32671887, 0.39553878, 0.45989826, 0.52009878, 0.5764598,
+            0.62917188, 0.67859507, 0.72482729, 0.76815531, 0.80879042,
+            0.84700373])#, 0.88317493])
```

Comment:
Is this number of sig figs going to bite us?

Reply:
nah, we match to a tolerance.